packages/entities/entities-plugins/src/components/PluginSelectCard.vue

```diff
@@ -0,0 +1,310 @@
+<template>
+  <KTooltip
+    class="plugin-card plugin-card-cursor-pointer"
+    :class="{
+      'disabled': !plugin.available || plugin.disabledMessage,
+    }"
+    :label="plugin.disabledMessage"
+    position-fixed
+  >
+    <KCard
+      class="plugin-card-content"
+      :data-testid="plugin.name"
+      :disabled="!plugin.available || plugin.disabledMessage"
+      has-hover
+      @click="isCustomPlugin ? undefined : noRouteChange ? emitPluginData() : handleCreateClick()"
+    >
+      <template
+        v-if="isCustomPlugin"
+        #statusHat
+      >
+        <div class="header-wrapper">
+          <KBadge v-if="!isCreateCustomPlugin">
+            {{ t('plugins.select.custom_badge_text') }}
+          </KBadge>
+        </div>
+      </template>
+
+      <template
+        v-if="isCustomPlugin && !isCreateCustomPlugin && !noRouteChange && controlPlaneId && (canDeleteCustom || canEditCustom)"
+        #actions
+      >
+        <KDropdownMenu
+          :kpop-attributes="{ placement: 'bottom-end' }"
+          width="150"
+        >
+          <template #default>
+            <KButton
+              class="actions-trigger"
+              data-testid="overflow-actions-button"
+              size="small"
+            >
+              <template #icon>
+                <KIcon
+                  :color="KUI_COLOR_TEXT_NEUTRAL_STRONGER"
+                  icon="more"
+                  :size="KUI_ICON_SIZE_30"
+                />
```

Comment:
suggestion: can you import from `@kong/icons` instead of using `KIcon`? Applies to the whole PR whenever you have a `KIcon` implemented